_pages/en/strategy-communications.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Communications Strategy
+ref: strategy-communications
+lang: en
+status: posted
+sections: Work In Progress
+permalink: /strategy-communications.html
+---
+
+## Communications Strategy
+
+- [Endorsement](#Endorsement)
+- [Outreach](#outreach)
+- [Feedback](#feedback)
+
+### Endorsement
+
+To be official, strategies, actions and other initiatives, must be endorsed or approved by management (DG, CIO, DM, ..).
+The following are way to get things endorsed.
+
+#### CIOEC Endorsement
+
+Requires wider consultation and presentation to DG Committee.
+Use for quarterly strategies and actions as well as other changes that require wider consultation.
```

Comment:
I think quarterly strategies is what we would not need this process for, hence the use case for the delegated authority from the CIO to the DG of strategy

Reply:
Quarterly strategies will probably need to go through CIOEC regardless of who is sending the direction.  Having DG send it makes the sending part easier for us, but strategies need wider consultation.

But maybe there are other ways that could be listed in this document.

Reply:
Agree with @smellems , part of stakeholder identification and what they will need to hear from us. CIOEC may be general direction and so would approve the Quarterly Strategy approach. But may not require approving individual bite-size one. Just the approach.